modules/overrides/config.go

```diff
@@ -247,5 +250,6 @@ func (c *Config) Collect(ch chan<- prometheus.Metric) {
 	ch <- prometheus.MustNewConstMetric(metricLimitsDesc, prometheus.GaugeValue, float64(c.Defaults.Read.MaxBlocksPerTagValuesQuery), MetricMaxBlocksPerTagValuesQuery)
 	ch <- prometheus.MustNewConstMetric(metricLimitsDesc, prometheus.GaugeValue, float64(c.Defaults.Global.MaxBytesPerTrace), MetricMaxBytesPerTrace)
 	ch <- prometheus.MustNewConstMetric(metricLimitsDesc, prometheus.GaugeValue, float64(c.Defaults.Compaction.BlockRetention), MetricBlockRetention)
+	ch <- prometheus.MustNewConstMetric(metricLimitsDesc, prometheus.GaugeValue, float64(c.Defaults.Compaction.CompactionWindow), MetricCompactionWindow)
```

Comment:
we seem to only expose some overrides here. perhaps only the ones a user may be interested in?

my guess is that we should not include this one, but i'm not 100% sure what this method is doing. perhaps determine its intent and add a comment?

Reply:
I was on the fence about this one.  I think the intent is to expose the config, but I somewhat wonder why metric a default value.  I think we can leave it out.